Entity/AddressEmbeddable.php

```diff
@@ -93,210 +95,67 @@ public function __toString(): string
         ]));
     }
 
-    /**
-     * @return string
-     */
-    public function getLocale(): ?string
-    {
-        return $this->locale;
-    }
-
-    /**
-     * @param string $locale
-     */
-    public function setLocale(string $locale = null): void
```

Comment:
Why are all the setters removed?

Reply:
I'm so convinced I saw it implemented in the base class, so I considered it a duplicated code. Don't ask me how it happened. I will restore them

Reply:
wait, they weren't removed

Reply:
Getters were removed as duplicates, but not the setters

Reply:
Indeed, getters, misread in the diff view. Didn't know they were overriding.